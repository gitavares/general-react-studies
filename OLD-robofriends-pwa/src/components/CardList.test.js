import { shallow } from 'enzyme';
import React from 'react';
import CardList from './CardList';

it('expect to render CardList component', () => {
    // expect(shallow(<Card />)).toMatchSnapshot();
    const mockRobots = [
        {
            id: 1,
            name: 'John Snow',
            username: 'JohnJohn',
            email: 'john@gmail.com'
        }
    ]
    const component = shallow(<CardList robots={mockRobots} />);
    expect(component.debug()).toMatchSnapshot();
})