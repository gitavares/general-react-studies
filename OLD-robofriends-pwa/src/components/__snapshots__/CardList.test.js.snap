// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CardList component 1`] = `
"<div>
  <Card id={1} name=\\"John Snow\\" email=\\"john@gmail.com\\" />
</div>"
`;
